portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,54 @@
+body {
+    font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+}
+
+#navbar {
+    text-align: center;
+}
+
+.name {
+    display: inline-block;
+    padding: 0px 10px 0px 10px;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    text-align: center;
+}
+
+#left-content {
+    text-align: center;
+    display: inline-block;
```

Comment:
Another optional style thing: the Google CSS style guide recommends indenting things by two spaces, and it recommends alphabetizing CSS declarations: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

I actually personally disagree with that last rule, so I never do it in my personal code. (I do have to follow it when I write code for my job though!) Again, happy to leave it up to you how strictly you want to follow the style guide.